test/assertions/arrayLength.js

```diff
@@ -0,0 +1,21 @@
+var testsRunningInNode = (typeof global !== "undefined" ? true : false),
+    chakram = (testsRunningInNode ? global.chakram : window.chakram),
+    expect = (testsRunningInNode ? global.expect : window.expect);
+
+describe("Chakram Assertions", function () {
+
+    describe("arrayLength()", function () {
+
+        it("should check length of array at body root", function () {
+            var response = chakram.get("https://jsonplaceholder.typicode.com/users");
```

Comment:
I found this endpoint which returns an array as the response body. I was not sure of how to get a similar response with httpbin.